test/random/TestInverseTransformSampling.cpp

```diff
@@ -0,0 +1,121 @@
+#include <Kokkos_MathematicalConstants.hpp>
```

Comment:
General comment, howe do I see that the test is passing/failing?

Reply:
One way is to compare the moments of generated samples with the analytic expression. I updated the test cases to write the error in a file.